aiohttp/web_urldispatcher.py

```diff
@@ -894,3 +908,85 @@ def freeze(self):
         super().freeze()
         for resource in self._resources:
             resource.freeze()
+
+    def add_routes(self, routes):
+        """Append routes to route table.
+
+        Parameter should be a sequence of RouteDef objects.
+        """
+        # TODO: add_table maybe?
+        for route in routes:
+            route.register(self)
+
+
+def route(method, path, handler, **kwargs):
+    return RouteDef(method, path, handler, kwargs)
+
+
+def head(path, handler, **kwargs):
+    return route(hdrs.METH_HEAD, path, handler, **kwargs)
```

Comment:
It would be good in general but for this particular case it doesn't make sense I think.
`hdrs` is not a part of our public API anyway.